cognite/neat/rules/models/wrapped_entities.py

```diff
@@ -91,39 +85,78 @@ def __repr__(self) -> str:
 
 class DMSFilter(WrappedEntity):
     @abstractmethod
-    def as_filter(self, default: Any | None = None) -> dm.filters.Filter:
+    def as_dms_filter(self, default: Any | None = None) -> dm.filters.Filter:
         raise NotImplementedError
 
+    @classmethod
+    def from_dms_filter(cls, filter: dm.Filter) -> "DMSFilter":
+        dumped = filter.dump()
+        if (body := dumped.get(dm.filters.Equals._filter_name)) and (value := body.get("value")):
```

Comment:
hard to read, cramming everything into single line is hard. I see why you opted for this (i.e. to figure out which filter is used), maybe body is not the right name , probably `filter_definition` is more appropriate or something along that line.

Reply:
Add to tech depth issue to make this part of code base a bit more readable.

Reply:
Maybe I am blind, but is this really that hard to read? 

One of the complicating matters is that the `cognite-sdk` filter does not have any public properties so you have to dump it, unless you want to risk it suddenly breaking.